lib/ansible/plugins/action/copy.py

```diff
@@ -26,9 +26,9 @@
 import traceback
 
 from ansible import constants as C
-from ansible.errors import AnsibleError, AnsibleActionFail, AnsibleFileNotFound
+from ansible.errors import AnsibleError, AnsibleFileNotFound, AnsibleActionFail, _AnsibleActionDone
```

Comment:
Should `_AnsibleActionDone` be public API?

Reply:
probably, if we ever finally define it ...